pandas/tests/extension/base/methods.py

```diff
@@ -164,6 +164,15 @@ def test_combine_add(self, data_repeated):
             orig_data1._from_sequence([a + val for a in list(orig_data1)]))
         self.assert_series_equal(result, expected)
 
+    @pytest.mark.xfail(reason="GH-24147", strict=True)
```

Comment:
This is a test for the original bug from the issue. Unfortunately, it doesn't run yet since `Series.where` loses the extension dtype. #24147 is fixing that.